README.md

```diff
@@ -17,55 +19,25 @@ npm install commerce-sdk-isomorphic
 
 ### Usage
 
-> **Note:** These are required parameters.
-
-| Parameter      | Description                                                                |
-| -------------- | :------------------------------------------------------------------------- |
-| clientId       | ID of the client account created with Salesforce Commerce.                 |
-| organizationId | The unique identifier for your Salesforce identity.                        |
-| shortCode      | Region-specific merchant ID.                                               |
-| siteId         | Name of the site to access data from, for example, RefArch or SiteGenesis. |
-
-
-### Configure the Isomorphic SDK
-
-
 ```javascript
```

Comment:
Simplified this code example, removing edge case for private client login and focusing on the specific calls.